hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -387,12 +387,16 @@ public final class OMConfigKeys {
    */
   public static final String OZONE_SNAPSHOT_DEEP_CLEANING_ENABLED = "ozone.snapshot.deep.cleaning.enabled";
   public static final boolean OZONE_SNAPSHOT_DEEP_CLEANING_ENABLED_DEFAULT = false;
+  @Deprecated
```

Comment:
can you add a javadoc here saying the following few static variables are deprecated because snapshot directory service is replaced by directory deleting service.

Reply:
done